### PR TITLE
Assignment for Select Quote

### PR DESCRIPTION
**Pull request by Jewel-Mir**
*Tested Search input feature.
*Tested Home Insurance button.
*Tested Title and Get a Free Quote button within the banner container at Home Insurance page.
*After clicking on Get a Free Quote button, verified a new url.
*Created a new Medicare page and tested the drop down for Plans button and selected a hidden title within.

### CODE REVIEW

File: test/specs/home.spec.js [hunk 1]
```diff
@@ -2,12 +2,61 @@ import { HomePage } from '../pageobjects/home.page'
...
-        browser.url('https://homeandauto.selectquote.com/')
+  
+   it('should redirect from https://homeandauto.selectquote.com/ to https://homeandauto.selectquote.com/insurance-overview/', () => {
+        browser.url("https://homeandauto.selectquote.com/")
         homePage.overviewButton.click()
 
         const expected = "https://homeandauto.selectquote.com/insurance-overview/"
         const value = browser.getUrl()
         expect(value).toEqual(expected)
+        browser.pause(3000);
```
Comment (sumitbasra) [inline test/specs/home.spec.js]: Please update all tests in this spec file without using pause. Take a look at waitFor methods in webdriverIO docs.

File: test/specs/medicare.spec.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+   it('should move to Plans and click on "Type of Plan Needed" button',() => {
+     medicare.plansButton.moveTo()
+     browser.pause(3000)
+     medicare.typeOfPlanNeeded.moveTo()
+     medicare.typeOfPlanNeeded.click()
+     browser.pause(3000)
+   
+   })
+})
```
Comment (sumitbasra) [inline test/specs/medicare.spec.js]: Always add one empty line at the end of the file.

File: test/specs/medicare.spec.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+describe('medicare selectquote home page',  () => {
+  
+   it('should open medicare home page', () => {
+        browser.url("https://medicare.selectquote.com/")
+       
+        const expected = "https://medicare.selectquote.com/"
+        const value = browser.getUrl()
+        expect(value).toEqual(expected)
+        browser.pause(3000);
```
Comment (sumitbasra) [inline test/specs/medicare.spec.js:12]: same as above. remove pause.

**Review by sumitbasra — CHANGES_REQUESTED** [review]
Please update. We should not be using pauses. Only in the special cases where you cant use any wait methods.

**Comment by Jewel-Mir** [comment]
Hi Sumit,

I have submitted my corrections. Please let me know if there is anything else I need to update. 
 Would you like me to open a new PR?

Thank you so much.

Sincerely
Jewel

**Comment by Jewel-Mir** [comment]
Hi Sumit,

I have made some additional corrections. Please let me know what you think.
Thank you.

Sincerely
Jewel

**Pull request closed without merging**